numberOfSubMaps: 1
poi:
  submap_0:
    party_room: [submap_0, party_room, -4.980, -6.084, 0.119]
    storage_room: [submap_0, storage_room, -11.192, -5.540, 1.448]
    leave_room: [submap_0, leave_room, 0.823, -8.587, 3.112]

object:
  information:
    coke: [information, coke, beverage]
    pringles: [information, pringles, snack]

  beverage:
    kitchen_counter: [beverage, kitchen_counter, 0.90]
    dinner_table: [beverage, dinner_table, 0.00]

  snack:
    kitchen_counter: [beverage, kitchen_counter, 0.90]
    dinner_table: [beverage, dinner_table, 0.00]

